config/initializers/ldap_instrumentation.rb

```diff
@@ -1,8 +1,8 @@
+require 'foreman/telemetry'
```

Comment:
This is not needed, it is already required in `5_telemetry`, which is guaranteed by the number to go prior this.